browser/brave_webtorrent/BUILD.gn

```diff
@@ -0,0 +1,23 @@
+import("//brave/components/brave_webtorrent/browser/buildflags/buildflags.gni")
+
+assert(enable_brave_webtorrent)
+
+source_set("brave_webtorrent") {
```

Comment:
why don't we make `sources.gni` for this one as well?

Reply:
because we don't want to use sources.gni unless we have to. See https://github.com/brave/brave-core/pull/8636/files#r625269687
Anything that doesn't have a circular dependency with chromium code should ideally be in its own target so we can better manage deps